workmail-save-and-update-email/template.yaml

```diff
@@ -0,0 +1,206 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description:
+    "WorkMail Save And Update Email"
+
+Parameters:
+    Disclaimer:
+        Type: String
+        Default: ''
+        Description: "[Optional] Text that you'd like to prepend to the email body."
+    Footer:
+        Type: String
+        Default: ''
+        Description: "[Optional] Text that you'd like to append to the email body. Use {key} to template the S3 object key for the saved message."
+    SubjectTag:
+        Type: String
+        Default: ''
+        Description: "[Optional] Text that you'd like to prepend to the email subject. Use {key} to template the S3 object key for the saved message."
+    SavedBucketExpiration:
+        Type: Number
+        Default: '1'
+        Description: "[Optional] Number of days to keep the saved messages in the S3 bucket. Defaults to 1."
+    SaveInternalMessages:
+        Type: String
+        Default: ''
+        Description: "[Optional] Determines if internal messages should be saved."
+    SaveExternalMessages:
+        Type: String
+        Default: 'True'
+        Description: "[Optional] Determines if external messages should be saved."
+    UpdateInternalMessages:
+        Type: String
+        Default: ''
```

Comment:
1. Both `UpdateInternalMessages` and `UpdateExternalMessages` parameters should only accept `True/False` as the value, therefore it's a good idea to add [AllowedValues](https://github.com/aws-samples/amazon-workmail-lambda-templates/blob/master/workmail-chat-bot-python/template.yaml#L9) property which will enforce that the values can only be `True` or `False`.
2. Like `UpdateExternalMessages` we should set the default value here as well right? I think we can set default to`False` here since most customers would only want to modify external messages, if anyone wants to modify internal messages they can set it to True here.

Reply:
Is there a best practice for handling boolean CFN parameters and python?

Reply:
So I'm not a CFN expert but I couldn't find `Boolean` parameter type in CloudFormation and piggybacking on `String` with two default values represents a Boolean parameter to the closest.

I'm not aware of a better way. Let me know if you find better ways :)